starfish/pipeline/registration/__init__.py

```diff
@@ -1,5 +1,6 @@
 import argparse
```

Comment:
I needed @ttung 's registration changes. These changes duplicate those of another PR. Can ignore this file, and I will fix the impending merge conflicts when the other branches get merged.